arewegleanyet.js

```diff
@@ -221,6 +229,26 @@ async function processRelease() {
       }
     }
 
+    let environmentProbes = 0;
```

Comment:
I would prefer the default value to be set here and the comment to be above this line